rllib/agents/dqn/learner_thread.py

```diff
@@ -54,6 +54,7 @@ def step(self):
                         td_error = info.get(
                             "td_error",
                             info[LEARNER_STATS_KEY].get("td_error"))
+                        replay.policy_batches[pid].set_get_interceptor(None)
```

Comment:
what does this do?

Reply:
I'll comment. It switches off lazy auto-conversion from numpy (SampleBatch) to torch tensors, which was the main reason for this bug.